modules/kube-audit/variables.tf

```diff
@@ -94,10 +94,10 @@ variable "audit_webhook_listener_image" {
 variable "audit_webhook_listener_image_version" {
   type        = string
   description = "The tag or digest for the audit webhook listener image to deploy. If changing the value, ensure it is compatible with `audit_webhook_listener_image`."
-  nullable    = false
-  default     = "deaabcb8225e800385413ba420cf3f819d3b0671@sha256:acf123f4dba63534cbc104c6886abedff9d25a22a34ab7b549ede988ed6e7144" # See, https://github.ibm.com/GoldenEye/issues/issues/13371
```

Comment:
Why is this change being made?? I think its better to have the variable with a default value so consumers understand what the default is. Otherwise they have to try find it in the code. Consumers of DA will never be able to know what default is